### PR TITLE
Expose SetStrongholdPasswordClearInterval

### LINKED ISSUE #1076: [Task]: Expose set stronghold password clear interval
### Task description

Expose set stronghold password clear interval

### Requirements

N/A

### Acceptance criteria

N/A

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Expose SetStrongholdPasswordClearInterval

## Links to any relevant issues

Fixes https://github.com/iotaledger/wallet.rs/issues/1076

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Thoralf-M** [comment]
https://github.com/iotaledger/iota.rs/pull/1005 needs to be merged before

**Review by samuel-rufi — DISMISSED** [review]

**Review by samuel-rufi — APPROVED** [review]

File: src/account_manager/operations/stronghold.rs [hunk 1]
```diff
@@ -23,6 +25,14 @@ impl AccountManager {
         Ok(())
     }
 
+    /// Sets the Stronghold password clear interval
+    pub async fn set_stronghold_password_clear_interval(&self, timeout: Option<Duration>) -> crate::Result<()> {
```
Comment (thibault-martinez) [inline src/account_manager/operations/stronghold.rs:29]: Why is this an Option ?

Comment (Thoralf-M) [reply src/account_manager/operations/stronghold.rs:29]: If it's `None` there will be no timeout and the key will never get cleared

Comment (thibault-martinez) [reply src/account_manager/operations/stronghold.rs:29]: Thanks

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**